index.html

```diff
@@ -1,343 +1,366 @@
-                    <img src="Images/Main/Social/Angelist.svg" alt="Angelist-mainpage">
-                </a>
-            </li>
-            <li>
-                <a href="www.twitter.com" class="soc-icon">
-                    <img src="Images/Main/Social/Twitter.svg" alt="twitter-mainpage">
-                </a>
-            </li>
-            <li>
-                <a href="https://medium.com/" class="soc-icon">
-                    <img src="Images/Main/Social/Medium.svg" alt="medium-mainpage">
-                </a>
-            </li>
 
+        <ul class="social-icon">
+          <li>
+            <a href="https://github.com/ulises2607" class="soc-icon">
+              <img src="Images/Main/Social/GitHub.svg" alt="github-profile" />
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/cesar-ulises-cappa-subelza/"
+              class="soc-icon"
+            >
+              <img
+                src="Images/Main/Social/Linkedin.svg"
+                alt="linkedin-profile"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.angellist.com/" class="soc-icon">
+              <img src="Images/Main/Social/Angelist.svg" alt="angelist.com" />
+            </a>
+          </li>
+          <li>
+            <a href="www.twitter.com" class="soc-icon">
+              <img src="Images/Main/Social/Twitter.svg" alt="twitter.com" />
+            </a>
+          </li>
+          <li>
+            <a href="https://medium.com/" class="soc-icon">
+              <img src="Images/Main/Social/Medium.svg" alt="medium.com" />
+            </a>
+          </li>
         </ul>
-        
-
+      </div>
     </section>
 
     <section class="recent-works">
-
-        
-
-        <div class="recent-w-title">
-            <h1>My Recent Works</h1>
-
-            <img src="Images/Work/Desktop/Vector.png" alt="">
-        </div>
-
-
-        <div class="container">
-
-            <img class="img-work" src="Images/Work/Work-img-1.png" alt="Multi-post-stories">
-        
-            <div class="mult-post">
-
-                <h2>Multi-Post Stories</h2>
-
-                <p>
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
-                    has been the industry's standard dummy text ever since the 1500s, when an unknown printer took 
-                    a standard dummy text.
-                </p>
-
-                <ul class="tech">
-                    <li class="tech-btn"><a href="#">CSS</a></li>
-                    <li class="tech-btn"><a href="#">HTML</a></li>
-                    <li class="tech-btn"><a href="#">Bootstrap</a></li>
-                    <li class="tech-btn"><a href="#">Ruby</a></li>
-                </ul>
-
-                <a href="#" class="btn-works-1">See Projects</a>
-            </div>
+      <div class="recent-w-title">
+        <h1>My Recent Works</h1>
```

Comment:
Great work so far! 👏  It appears that you're using the `<h1>` tag for each section title. However, it's recommended to have only one `<h1>` tag per webpage for better accessibility. Consider replacing the `<h1>` tag with an appropriate heading tag, such as `<h2>` to `<h6>`, depending on the hierarchy and relevance of the titles within your code. You can refer to the [documentation on heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility_concerns) for more information.

By implementing these suggestions, you'll ensure that your design is consistent and adheres to best practices. Keep up the excellent work! Happy coding! 😊

**_Note that most of the headings in your webpage are not structured well. please consider the same requirement for all headings._**